dependencies.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Don't know how new it is, but this file tells Package Control, which packages we depend on. It will than install them automatically. Dependencies are special packages which can contain libraries and such stuff.